### ISSUE TITLE
Add support for putting `sanitize` tags in XML comments (`<!-- -->`)

### ISSUE THREAD

**Issue by LinusOstlund** [repobee/repobee-sanitizer#143]
I have put my sanitizer tags in XML-style (see below):

```
> **Student's Answer:** 
<!--REPOBEE-SANITIZER-START-->

| Value     | Answer                 |
|-----------|------------------------|
| `0`       | int                    |
| `"hello"` | String                 |
| `101`     | int                    |
| `-1`      | int                    |
| `true`    | boolean                |
| `"33"`    | String                 |
| `3.1415`  | double (or float)      |

<!--REPOBEE-SANITIZER-REPLACE-WITH-->

| Value     | Answer                 |
|-----------|------------------------|
| `0`       | WRITE YOUR ANSWER HERE |
| `"hello"` | ...                    |
| `101`     |                        |
| `-1`      |                        |
| `true`    |                        |
| `"33"`    |                        |
| `3.1415`  |                        |

<!--REPOBEE-SANITIZER-END-->
```

This issues the following error when attempting the recommended `repobee sanitize repo --target-branch master` workflow:

```
Syntax errors detected in 1 file(s):

docs/README.md
    Line 84: Missing prefix: <!--
    Line 85: Missing prefix: <!--
    Line 86: Missing prefix: <!--
    Line 87: Missing prefix: <!--
    Line 88: Missing prefix: <!--
    Line 89: Missing prefix: <!--
    Line 90: Missing prefix: <!--
    Line 91: Missing prefix: <!--
    Line 92: Missing prefix: <!--
    Line 93: Missing prefix: <!--
    Line 94: Missing prefix: <!--
```

I suggest adding support for XML comments.

**Comment by tohanss** [comment]
The error you receive is intended functionality, if a prefix (or comment) is used for the markers, the lines inside the replace block must also have the prefix. This is to ensure the code can be compiled, and that only one of the versions is compiled per main/solutions branch. However, XML works differently, so perhaps that is what you want, to not have to comment out the replace block? 

* In that case I can probably add a flag to continue despite this error. 
* Alternatively, I could also add this to the `--force` flag, but I will have to think about that usage

This also shows another issue we had hoped to avoid
* XML _only_ uses multi-line comments, which we do not support, meaning you will be left with a `-->` after every commented line in the replace block should you choose to go for that approach, adding some manual labor (ugh). As for the markers, the whole line is removed, so that is not a problem.

**To summarise** 
Let me know if you wish for the code to work as you posted in the issue or if you would be fine with commenting out the lines inside the replace block, either way i will implement the necessary changes to make that work :)
